pkg/openstack/loadbalancer.go

```diff
@@ -549,7 +549,7 @@ func (lbaas *LbaasV2) createFullyPopulatedOctaviaLoadBalancer(name, clusterName
 		svcConf.lbMemberSubnetID = loadbalancer.VipSubnetID
 	}
 
-	if err := openstackutil.WaitLoadbalancerActive(lbaas.lb, loadbalancer.ID); err != nil {
+	if loadbalancer, err = openstackutil.WaitLoadbalancerActive(lbaas.lb, loadbalancer.ID); err != nil {
```

Comment:
I would prefer a quick update on the function name .. will submit a nit PR for this

Reply:
You probably mean to make it `LoadBalancer`?

Reply:
openstackutil.WaitLoadbalancerActive  => openstackutil.WaitLoadbalancerActiveAndGetLB